create section "price" with basic styles

HannaSydoruk opened the pull request.

HannaSydoruk committed:
create section with basic styles

HannaSydoruk committed:
Update price.css

css animation is added

andreasrukom19 reviewed (changes requested):
Все добре. Але у всіх секціях контент повинен бути обгорнутий у div.container

andreasrukom19 reviewed (commented):
Потрібне доопрацювання

HannaSydoruk committed:
div.container is added

div.container is added

andreasrukom19 commented on src/css/price.css:
```diff
@@ -0,0 +1,99 @@
+.container {
+  width: 100%;
+  padding: 64px 16px;
```
Скоріш за все `padding-bottom: 64px` тут не потрібен. Так як у наступної секції повинен бути padding-top: 64px

andreasrukom19 commented on src/css/price.css:
```diff
@@ -0,0 +1,99 @@
+.price-item-name {
+  font-size: 28px;
+  font-weight: 400;
+  text-transform: uppercase;
```
Треба обсудити font-family. Шрифти то підключені у index.html, але ж для body глобально нічого не прописано

andreasrukom19 commented on src/css/price.css:
```diff
@@ -0,0 +1,99 @@
+@media screen and (min-width: 375px) {
+  .container {
+    max-width: 375px;
+    margin: auto;
```
Повинно бути `margin: 0 auto`

andreasrukom19 commented on src/css/price.css:
```diff
@@ -0,0 +1,99 @@
+@media screen and (min-width: 768px) {
+  .container {
+    max-width: 704px;
+    padding: 100px 32px;
```
Знову ж таки, нижній padding не ставимо

andreasrukom19 commented on src/css/price.css:
```diff
@@ -0,0 +1,99 @@
+
+@media screen and (min-width: 1440px) {
+  .container {
+    max-width: 1096px;
```
Повинно бути max-width: 1440px;

andreasrukom19 commented on src/css/price.css:
```diff
@@ -0,0 +1,99 @@
+@media screen and (min-width: 1440px) {
+  .container {
+    max-width: 1096px;
+    padding: 100px 172px;
```
padding-bottom не потрібний

andreasrukom19 commented on src/css/price.css:
```diff
@@ -0,0 +1,99 @@
+  font-style: normal;
+  line-height: 1.5;
+  border-top: 1px solid var(--blue-font);
+  box-sizing: border-box;
```
До проекту підключений normalise. Так що box-sizing там прописаний

andreasrukom19 commented on src/css/price.css:
```diff
@@ -0,0 +1,99 @@
+
+.price-item:last-child {
+  border-bottom: 1px solid var(--blue-font);
+  box-sizing: border-box;
```
Аналогічно, box-sizing не потрібний

HannaSydoruk replied:
Андрію, виправила по зауваженням - згодна в принципі. шрифт поки залишила, тому що у боді його не прописано, також якщо перенесеться в загарльні стилі visually-hidden, його теж можна буде прибрати з локальних стилів. але поки залишаю

HannaSydoruk committed:
виправлені padding, margin, видалено box-sizing

HannaSydoruk committed:
Update price.html

додано доп.клас price-container для специифічних паддінгів секції

HannaSydoruk committed:
додатковий контейнер

додатковий контейнер

andreasrukom19 commented on src/css/price.css:
```diff
@@ -0,0 +1,97 @@
+.price-container {
+  width: 100%;
```
Потрібно видалити

andreasrukom19 commented on src/css/price.css:
```diff
@@ -0,0 +1,97 @@
+.price-container {
+  width: 100%;
+  padding: 64px 16px 0 16px;
```
Тільки padding-top: 64px

andreasrukom19 commented on src/css/price.css:
```diff
@@ -0,0 +1,97 @@
+  clip-path: inset(100%);
+  clip: rect(0 0 0 0);
+  overflow: hidden;
+}
```
Властивості для  visually-hidden будуть прописані в загальному файлі стилів. Потрібно буде видалити

andreasrukom19 commented on src/css/price.css:
```diff
@@ -0,0 +1,97 @@
+.price-item {
+  padding: 30px 0;
+  color: var(--blue-font);
+  font-family: Roboto;
```
font-family прописаний для body в загальному файлі.

andreasrukom19 commented on src/css/price.css:
```diff
@@ -0,0 +1,97 @@
+    max-width: 375px;
+    margin: 0 auto;
+  }
+}
```
Даний медіазапит не потрібний

andreasrukom19 commented on src/css/price.css:
```diff
@@ -0,0 +1,97 @@
+  .price-container {
+    max-width: 768px;
+    padding: 100px 32px 0 32px;
+  }
```
Тільки padding-top: 100px; Я б ці відступи взагалі задавав би секціям (верхній паддінг)

andreasrukom19 commented on src/css/price.css:
```diff
@@ -0,0 +1,97 @@
+@media screen and (min-width: 1440px) {
+  .price-container {
+    max-width: 1440px;
+    padding: 100px 172px 0 172px;
```
Те саме. padding-top: 100px;

HannaSydoruk replied:
тут залишаю padding: 100px 172px 0 172px; тому що секція вужча, ніж більшість інших

HannaSydoruk committed:
fixed

видалені класи і властивості, які мають бути винесені в загальні (згідно коментарів)
